src/pages/UploadPage.tsx

```diff
@@ -58,59 +64,14 @@ export default function UploadPage() {
     setSelectedVacancy(event.target.value);
   };
 
-  const isSendButtonDisabled = !cover_letter || selectedVacancy === '';
+  const isSendButtonDisabled = !cv || selectedVacancy === '';
 
   return (
     <Container maxWidth="lg" sx={{ mt: 10, mb: 4 }}>
       <h1>File Upload</h1>
       <h4>CV</h4>
-      <Box sx={{ display: 'flex', mb: 2, flexDirection: 'row' }}>
-        <TextField
-          disabled
-          id="cover_letter-input"
-          label="CV"
-          defaultValue="Insert File here"
-          size="small"
-          sx={{ width: '300px', marginRight: 2 }}
-          value={cover_letter?.name}
-        />
-        <Button
-          size="small"
-          variant="contained"
-          component="label"
-          color="secondary"
-          sx={{ width: '100px', height: '40px', textTransform: 'capitalize' }}
-          disableElevation
-        >
-          Select File
-          <input
-            type="file"
-            accept="application/pdf"
-            hidden
-            onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
-              if (!event.target.files) return;
-              setCoverLetter(event.target.files[0]);
-            }}
-          />
-        </Button>
-      </Box>
       <Box sx={{ display: 'flex', flexDirection: 'row', mb: 2 }}>
-        <FormControl sx={{ width: 300 }} size="small">
-          <InputLabel id="vacancy-label">Vacancy</InputLabel>
-          <Select
```

Comment:
Please do not delete the drop down. The endpoint need both the CV and the selected vacancy to rate the applicant